specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/applications.json

```diff
@@ -253,7 +253,7 @@
         },
         "subDomainSuffix": {
           "type": "string",
-          "description": "The subDomainSuffix of the application and can not greater than 3 characters."
+          "description": "The subDomainSuffix of the application. It cannot be greater than 3 characters."
```

Comment:
Does it make sense to use `minLength`/`maxLength` instead of documentation alone?

Reply:
> Does it make sense to use `minLength`/`maxLength` instead of documentation alone?
Thanks for your comment. And your comment makes sense to me. However, we think that we should not add the restriction. 

After discussion we decide to remove it from description. I have updated the PR. Thank you.

Reply:
No problem